cmd/unused-exporter/exporter.go

```diff
@@ -150,13 +150,11 @@ func (e *exporter) Collect(ch chan<- prometheus.Metric) {
 			for _, d := range disks {
 				m := d.Meta()
 
-				var ts float64
-				lastUsed := d.LastUsedAt()
-				if lastUsed.IsZero() {
-					ts = 0.0
-				} else {
-					ts = float64(lastUsed.UnixMilli())
-				}
+ts := 0.0
+lastUsed := d.LastUsedAt()
+if !lastUsed.IsZero() {
+    ts = float64(lastUsed.UnixMilli())
+}				
```

Comment:
@dannykopping looks like the suggestion wasn't formatted properly 😭

Reply:
I has spaces at the end! Fire up your Emacs, `M-x delete-trailing-whitespace` et voilà! Or just add a `before-save-hook` that runs it automatically 😬